appOPHD/UI/FileIo.h

```diff
@@ -25,7 +27,8 @@ class FileIo : public Window
 	FileIo();
 	~FileIo() override;
 
-	void setMode(FileOperation fileOp);
+	void showOpen(const std::string& directory = constants::SaveGamePath);
+	void showSave(const std::string& directory = constants::SaveGamePath);
```

Comment:
Kind of a similar concern about the putting `constants::SaveGamePath` as default value in `FileIo.h` as the previous comment.

The current use of this dialog is for saved games. I'd be hesitant to say it's specific to that though.